content/docs/gke/deploy/reasons.md

```diff
@@ -1,29 +1,39 @@
 +++
-title = "Reasons to Use Kubeflow on GCP"
-description = "Reasons to deploy Kubeflow on GCP"
-weight = 8
+title = "Features of Kubeflow on GCP"
+description = "Reasons to use Kubeflow on Google Cloud Platform (GCP)"
+weight = 10
 +++
 
-Running Kubeflow on [GKE](https://cloud.google.com/kubernetes-engine/docs)
-brings the following advantages:
+Running Kubeflow on GCP brings you the following features:
 
   * You use
     [Deployment Manager](https://cloud.google.com/deployment-manager/docs/) to
     declaratively manage all non-Kubernetes resources (including the GKE 
     cluster). Deployment Manager is easy to customize for your particular use
     case.
-  * You can take advantage of GKE autoscaling to scale your cluster horizontally 
+  * You can take advantage of 
+    [GKE](https://cloud.google.com/kubernetes-engine/docs) autoscaling to scale 
+    your cluster horizontally 
     and vertically to meet the demands of machine learning (ML) workloads with 
     large resource requirements.
   * [Cloud Identity-Aware Proxy (Cloud IAP)](https://cloud.google.com/iap/) 
     makes it easy to securely connect to Jupyter and other
     web apps running as part of Kubeflow.
-  * Basic auth service supports simple username/password access to your 
-    Kubeflow. It is an alternative to Cloud IAP service:
-    * We recommend IAP for production and enterprise workloads.
-    * Consider basic auth only when trying to test out Kubeflow and use it 
+  * Kubeflow's basic authentication service supports simple username/password
+    access to your Kubeflow resources. Basic auth is an alternative to Cloud
```

Comment:
Should this reference to 'basic auth' change to "basic authentication"?

Reply:
I decided to leave "basic auth" in subsequent mentions, after using the full term once in this section, given that "basic auth" is a fairly well known term. Happy to change it if you think it'd be clearer, perhaps in a subsequent PR?

Reply:
I defer to your judgment! I just wanted to make sure that this was intentional.

Reply:
Thanks :) I'll leave it as it is for now.